File: day1/plus-minus/solution.js
function plusMinus(arr) {
  let negative = arr.reduce((acum, value) => {
    if (value < 0) {
      return acum + 1;
    }
    return acum;
  }, 0);
  let zero = arr.reduce((acum, value) => {
    if (value === 0) {
      return acum + 1;
    }
    return acum;
  }, 0);
  let positive = arr.reduce((acum, value) => {
    if (value > 0) {
      return acum + 1;
    }
    return acum;
  }, 0);
  console.log(
    (positive / arr.length).toFixed(6),
    '\n',
    (negative / arr.length).toFixed(6),
    '\n',
    (zero / arr.length).toFixed(6)
  );
}

File: day1/plus-minus/solution.py
def plusMinus(arr):
    negative =0
    zero = 0
    positive =0
    length = len(arr)
    for value in arr:
        if value < 0:
            negative +=1
        elif value == 0:
            zero += 1
        else:
            positive += 1
    print(f'{positive/length:.6f}', '\n', f'{negative/length:.6f}', '\n', f'{zero/length:.6f}')